scripts/generate-artifacts.ps1

```diff
@@ -27,7 +27,7 @@ $storageAccount = (Get-AzStorageAccount -ErrorAction SilentlyContinue | Where-Ob
 if ($storageAccount) {
     Write-Output "Storage account found"
 } else { 
-    throw "Unable to find storage account $ArtifactsStorageAccountName . Please make sure, that you specified the correct name of the storage account for the artifacts and that it is present in your Azure subscription"
+    throw "Unable to find storage account $ArtifactsStorageAccountName. Please make sure, that you specified the correct name of the storage account for the artifacts and that it is present in your Azure subscription"
```

Comment:
The error message here is informative, but ensure that the storage account name is correctly passed to the script. Double-check the parameter `$ArtifactsStorageAccountName` to ensure it is correctly set and used.